VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.html

```diff
@@ -6,8 +6,8 @@
   </p>
   <hr />
   <a *ngIf="participant.isInHearing()" (click)="toggleMuteParticipant()">{{ getMutedStatusText() }}</a>
-  <a *ngIf="(participant.isWitness || participant.isQuickLinkObserver) && !participant.isInHearing()" (click)="callWitnessIntoHearing()">{{'judge-context-menu.call-witness' | translate}}</a>
-  <a *ngIf="(participant.isWitness || participant.isQuickLinkObserver) && participant.isInHearing()" (click)="dismissWitnessFromHearing()">{{'judge-context-menu.dismiss-witness' | translate}}</a>
+  <a *ngIf="callParticipantIntoHearing()" (click)="callWitnessIntoHearing()">{{'judge-context-menu.call-witness' | translate}}</a>
```

Comment:
Can we change callWitnessIntoHearing to reflect that it can be other types of participants?

Reply:
Done